ceph-dev-new-trigger/config/definitions/ceph-dev-new-trigger.yml

```diff
@@ -28,20 +28,46 @@
           wipe-workspace: true
 
     builders:
-      - shell:
-          !include-raw:
-            - ../../../scripts/build_utils.sh
-            - ../../build/notify
-      - trigger-builds:
-        - project: 'ceph-dev-new'
-          predefined-parameters: |
-            BRANCH=${GIT_BRANCH}
-            FORCE=True
-        - project: 'ceph-dev-new'
-          predefined-parameters: |
-            BRANCH=${GIT_BRANCH}
-            FORCE=True
-            FLAVOR=notcmalloc
+      - conditional-step:
```

Comment:
@alfredodeza i switched to "Conditional BuildStep Plugin". seems `combination-filter` does not allow me to access env variables. hopefully this is less scary, and works. but i am not sure if this plugin is installed in our jenkins. could you help me verify it?

Reply:
Ok so this will need that plugin installed. /cc @djgalloway

Reply:
@alfredodeza It's already installed.  Green light from me as far as that goes.